components/image/image.css

```diff
@@ -0,0 +1,11 @@
+/* || RESPONSIVE IMAGE */
+.img-responsive {
+  max-width: 100%;
+  height: auto;
+}
+
+/* || ROUND IMAGE */
+.img-round {
+  border-radius: 50%;
```

Comment:
instead of this make a class like .br-50 and give it the property of border-radius : 50% that way you can use it multiple places

Reply:
Ya i can do that for sure!  right now i'm configuring each component based on their own utility classes.